week3/leap_year.py

```diff
@@ -10,7 +10,11 @@
 
 def is_leap_year(year):
 
-    return True if year%4 == 0 and year%100 != 0 or year%400 == 0 else False
+    if year % 4 == 0 and year % 100 != 0 or year % 400 == 0:
+
+    	return True
+
+    return False
```

Comment:
that is good! then here i think it is just a question of taste, your solution was good too :)